Sources/SpeziOnboarding/OnboardingStack.swift

```diff
@@ -0,0 +1,18 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+// swiftlint:disable file_types_order
+
+import SpeziViews
+
+/// Managed, sequential presentation of consecutive ``OnboardingView``s
+public typealias OnboardingStack = ManagedNavigationStack
```

Comment:
Nit: we usually have two new lines between imports and definitions.

Reply:
can we somehow add a swiftlint rule to enforce this?

Reply:
I think Andi investigated this, sadly not possible.